### ISSUE TITLE
Stat Boost & Loadout Comparison Graph

### ISSUE THREAD

**Issue by RonnyTommy548** [weirdgloop/osrs-dps-calc#470]
### What went wrong?

When no stat boost (ex: ranging pot) is active, the dps at the top matches lvl 99 on the Loadout Comparison Graph at the bottom (X axis set to player ranged level and Y axis set to player damager-per-second).
However, when a stat boost (ex: ranging pot) is active, the dps at the top increases as you would expect, but then the Loadout Comparison Graph shows the same boosted DPS value at lvl 99, which makes no sense. It's almost as if the stat boost is getting applied twice on the Loadout Comparison Graph.

### What did you expect to happen?

Example: https://dps.osrs.wiki?id=EntangleGrimEathram

With no ranging potion active, the dps shown at the top is 3.386 and lvl 99 shows the same value (3.386) on the Loadout Comparison Graph for player ranged level on the X axis and player damage-per-second on the Y axis. Activating the ranging potion increases the dps to 3.999 at the top of the page, which is expected, however the Loadout Comparison Graph then also shows 3.999 dps for level 99 ranged, which doesn't make any sense, since that increased dps value was only possible from having a ranged level boosted beyond level 99. I would expect the value at the top to only match the graph at lvl 112, and for lvl 99 to be lower

### What browsers are you seeing the problem on?

Chrome

### What device(s) are you seeing the problem on?

Windows

### Any other information

TLDR: When a ranging pot is active, dps vs. ranged level drain on Loadout Comparison Graph is not accurate. "99" on that graph is not actually lvl 99, but rather 112
